### PR TITLE
[Isuue: #2554] clean code 

### PR DESCRIPTION
**Pull request by li-keguo**
about issue #2554

work list:
- [x] clean some warning code

### CODE REVIEW

File: shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-alibaba-dubbo/src/main/java/org/apache/shenyu/plugin/alibaba/dubbo/cache/AlibabaDubboConfigCache.java [hunk 1]
```diff
@@ -47,26 +48,24 @@
...
-
+    
     private RegistryConfig registryConfig;
-
+    
     private final LoadingCache<String, ReferenceConfig<GenericService>> cache = CacheBuilder.newBuilder()
             .maximumSize(Constants.CACHE_MAX_COUNT)
             .removalListener((RemovalListener<Object, ReferenceConfig<GenericService>>) notification -> {
                 ReferenceConfig<GenericService> config = notification.getValue();
                 if (config != null) {
```
Comment (qicz) [inline shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-alibaba-dubbo/src/main/java/org/apache/shenyu/plugin/alibaba/dubbo/cache/AlibabaDubboConfigCache.java:62]: use `Objects.nonNull`

File: shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-apache-dubbo/src/main/java/org/apache/shenyu/plugin/apache/dubbo/cache/ApacheDubboConfigCache.java [hunk 2]
```diff
@@ -48,26 +49,24 @@
...
-
+    
     private RegistryConfig registryConfig;
-
+    
     private final LoadingCache<String, ReferenceConfig<GenericService>> cache = CacheBuilder.newBuilder()
             .maximumSize(Constants.CACHE_MAX_COUNT)
             .removalListener((RemovalListener<Object, ReferenceConfig<GenericService>>) notification -> {
                 ReferenceConfig<GenericService> config = notification.getValue();
                 if (config != null) {
```
Comment (qicz) [inline shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-apache-dubbo/src/main/java/org/apache/shenyu/plugin/apache/dubbo/cache/ApacheDubboConfigCache.java:63]: use `Objects.nonNull`

File: shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-alibaba-dubbo/src/main/java/org/apache/shenyu/plugin/alibaba/dubbo/cache/AlibabaDubboConfigCache.java (hunk 1 above)
Comment (li-keguo) [reply shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-alibaba-dubbo/src/main/java/org/apache/shenyu/plugin/alibaba/dubbo/cache/AlibabaDubboConfigCache.java:62]: ok

File: shenyu-plugin/shenyu-plugin-api/src/main/java/org/apache/shenyu/plugin/api/result/ShenyuResultWrap.java [hunk 3]
```diff
@@ -25,6 +25,10 @@
  */
 public final class ShenyuResultWrap {
     
+    private ShenyuResultWrap() {
+
```
[the remark this answers is not in the document]
Comment (li-keguo) [reply shenyu-plugin/shenyu-plugin-api/src/main/java/org/apache/shenyu/plugin/api/result/ShenyuResultWrap.java:29]: ok

**Review by yu199195 — APPROVED** [review]

**Pull request merged**